pkg/client/unversioned/cache/listers.go

```diff
@@ -227,12 +227,12 @@ func (s *StoreToReplicationControllerLister) GetPodControllers(pod *api.Pod) (co
 }
 
 // StoreToDaemonLister gives a store List and Exists methods. The store must contain only Daemons.
-type StoreToDaemonLister struct {
+type StoreToDaemonSetLister struct {
 	Store
 }
 
 // Exists checks if the given dc exists in the store.
```

Comment:
Please make this replacement globally; I won't bother to mark each instance. (I found about six places searching for "dc " and " dc")